pkg/wakaama/contrib/objects/on_off_switch.c

```diff
@@ -0,0 +1,552 @@
+/*
+ * Copyright (C) 2024 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @{
+ * @ingroup     lwm2m_objects_on_off_switch
+ *
+ * @file
+ * @brief       On/off switch object implementation for LwM2M client using Wakaama
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ * @}
+ */
+
+#include "mutex.h"
+#include "inttypes.h"
+#include "liblwm2m.h"
+#include "lwm2m_client.h"
+#include "objects/on_off_switch.h"
+#include "ztimer.h"
+#include "ztimer/stopwatch.h"
+
+#define ENABLE_DEBUG 1
+#include "debug.h"
```

Comment:
```suggestion
#define ENABLE_DEBUG 0
#include "debug.h"
```

as proposed by the static tests.

Reply:
Missed that one, changed.